src/components/EventManagement/AddNewEvents.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+
+const AddNewEvents = () => {
+  const [title, setTitle] = useState("");
+  const [weddingDecription, setWeddingDecription] = useState("");
+  const [startTime, setStartTime] = useState(new Date());
+  const [endTime, setEndTime] = useState(new Date());
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const newDateTime = new Date();
+    setStartTime(newDateTime);
+    const newDateTimeEnd = new Date();
+    setEndTime(newDateTimeEnd);
+    console.log(title, weddingDecription, startTime, endTime);
+    try {
+      const postData = await fetch("http://localhost:3000/events", {
```

Comment:
please create an env file and add the API domain `http://localhost:3000/` in env and use `process.env` attributes to use the domain with API paths.

Reply:
Please create a new file in api directory and keep all the APIs in that file. Each API should be added in a function and use that api function in th component.
```
File Path: <app_root>/src/api/index.js
```

Reply:
done